kong/db/dao/plugins/go.lua

```diff
@@ -185,6 +228,37 @@ local get_field
 do
   local exposed_api = {
     kong = kong,
+    ["kong.log.serialize"] = function()
+      return cjson_encode(preloaded_stuff.basic_serializer or basic_serializer.serialize(ngx))
+    end,
+
+    ["kong.nginx.get_var"] = function(v)
+      return ngx.var[v]
+    end,
+
+    ["kong.nginx.get_tls1_version_str"] = ngx_ssl.get_tls1_version_str,
+
+    ["kong.nginx.get_ctx"] = function(v)
+      return ngx.ctx[v]
```

Comment:
We should instead be using `kong.ctx` as that contains extra logic - eg, for `kong.ctx.shared`.

Reply:
I originally added this `kong.nginx.get_ctx` in the Go PDK as a way to access directly the raw `ngx.ctx` because my initial line-by-line port of `file-log` needed it (when the equivalent of the all-in-one `serialize` function was implemented in Go itself). I think `kong.ctx` should be eventually exposed as `kong.ctx` instead, so I think this is good as is.